consensus/src/chained_bft/test_utils/mod.rs

```diff
@@ -192,6 +193,14 @@ pub fn placeholder_certificate_for_block(
     )
 }
 
+pub fn dummy_sync_info() -> SyncInfo {
```

Comment:
I roughly remember we replaced all the word `dummy` with `placeholder` before

Reply:
This is not placeholder though (like LI placeholder), its more like just some arbitrary value that does not matter for test...

Reply:
Yes, what Andrey says. We'd like to have different LedgerInfos (with different digests) for the tests, so we need some random arbitrary values there (unlike a placeholder, which is typically all zeroes and empty vectors).

Reply:
then probably `arbitrary_sync_info`?